inc/core/theme-block-patterns.php

```diff
@@ -30,23 +30,16 @@ function bcgov_blocks_theme_register_block_patterns() {
 	 */
 
 	$block_pattern_categories = array(
```

Comment:
it was great that you fixed all the inner arrays to the short form, you forgot the main array though.

Reply:
I will get there... Cub Scouts honour.